apps/src/applab/applab.js

```diff
@@ -1040,7 +1041,11 @@ Applab.init = function(config) {
     }
     var designModeButton = document.getElementById('designModeButton');
     if (designModeButton) {
-      dom.addClickTouchEvent(designModeButton, Applab.onDesignModeButton);
+      // Simulate a run button click, to load the channel id.
+      var designModeClick = studioApp.runButtonClickWrapper.bind(
```

Comment:
Yes, the plan is for this pattern to go away in the future: https://www.pivotaltracker.com/story/show/95128786